_data/nav.yml

```diff
@@ -20,8 +20,6 @@ pages:
             url: /about/
           - text: Security Working Group
             url: /swg/
-          - text: Committee
```

Comment:
Did you intend to remove this from the Navigation Bar ?

Reply:
Yes I did, we have no Committee in OP-TEE so it makes no sense to have it there.